README.md

```diff
@@ -7,22 +7,30 @@
 
 [![Build status](https://github.com/maticnetwork/avail-light/actions/workflows/default.yml/badge.svg)](https://github.com/maticnetwork/avail-light/actions/workflows/default.yml) [![Code coverage](https://codecov.io/gh/maticnetwork/avail-light/branch/main/graph/badge.svg?token=7O2EA7QMC2)](https://codecov.io/gh/maticnetwork/avail-light)
 
-![demo](./img/prod_demo.png)
+![demo](./img/lc.png)
 
 ## Introduction
 
-`avail-light` is a data availability light client that can do the following:
+`avail-light` is a data availability light client with the following functionalities:
 
-* Listen for finalised blocks.
-* Gain confidence for `N` *cells*, where *cell* is defined as a `{row, col}` pair.
-  > As soon as a finalised block is available, the light client attempts to gain confidence by asking for a proof from a full 
-  client via JSON-RPC.
+* Listening on the Avail network for finalized blocks
+* Random sampling and proof verification of a predetermined number of cells (`{row, col}` pairs) on each new block. After successful block verification, confidence is calculated for a number of *cells* (`N`) in a matrix, with `N` depending on the percentage of certainty the light client wants to achieve.
```

Comment:
just to make sure the number of cells for random sampling is still based on confidence provided right?

Reply:
Yes, yes, it was never changed.